nyc/utils.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import print_function
+import httplib2
+import os
+
+from apiclient import discovery
+from oauth2client import client
+from oauth2client import tools
+from oauth2client.file import Storage
+from icalendar import Calendar, Event
+
+from django.conf import settings 
+from django.http import HttpResponse
+
+from datetime import datetime, timedelta
+
+try:
+    import argparse
+    flags = tools.argparser.parse_args([])
+except ImportError:
+    flags = None
+
+# Helper function for exporting events to Google calendar.
```

Comment:
Tiny thing, but can you add these comments to the method docstring?

Reply:
Yup...I'll also remove the unnecessary comments and add a citation for the code.